Season4/TeamCode/src/main/java/org/firstinspires/ftc/team13180s3/MoveServo.java

```diff
@@ -30,7 +30,7 @@ public void execute(){
         if(time.milliseconds() >= TimeoutMs){
             state = State.STOP;
         }
-        //every five seconds move the servo
+        //every five seconds move the servo and change the position
         else if(time.milliseconds() >= 15000){
             servo.setPosition(1);
```

Comment:
sample code review comment: servo should be set to position 0 rather position 1.